src/Algolia.Search/Transport/RetryStrategy.cs

```diff
@@ -87,26 +88,25 @@ public IEnumerable<StatefulHost> GetTryableHost(CallType callType)
         /// Update host's state
         /// </summary>
         /// <param name="tryableHost">A statefull host</param>
-        /// <param name="httpResponseCode">An http response code</param>
-        /// <param name="isTimedOut">Is the request timed out</param>
+        /// <param name="response">Algolia's API response</param>
         /// <returns></returns>
-        public RetryOutcomeType Decide(StatefulHost tryableHost, int httpResponseCode, bool isTimedOut)
+        public RetryOutcomeType Decide(StatefulHost tryableHost, AlgoliaHttpResponse response)
         {
             lock (_lock)
             {
-                if (!isTimedOut && IsSuccess(httpResponseCode))
+                if (!response.IsTimedOut && IsSuccess(response.HttpStatusCode))
```

Comment:
Maybe the `IsSuccess` could also accept the entire response to make it consistence with `IsRetryable`.

Reply:
👍